packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -271,7 +271,7 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
     Color backgroundColor,
     Color activeColor,
     Color inactiveColor,
-    Size iconSize,
+    double iconSize,
```

Comment:
I am confused about how this ever worked.

I am also hoping this doesn't end up being a breaking change under our new breaking change policy (/cc @Hixie).

Reply:
To be a breaking change there should be some code that call this method with a `Size`. But call it with a `Size` would always fail. So I'll arg this is not a breaking change :)